abdasgupta/docker
# Enabling 'docker ps' on isolated container.

**- What I did**

Modified container/libvirt.go to produce minimal changes compared to the upstreamed docker. Depending on Container state, containers appear on docker ps list. These commits took care of container states for isolated container during container stop and container remove.

**- How I did it**

**- How to verify it**
Run a long running docker isolated container and see it in docker ps command. Run a short lived docker isolated container to see it in docker ps -a command after finishing. Stop a long running isolated container to see it in docker ps -a.
**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
